connect/src/main/java/com/stripe/android/connect/analytics/ConnectAnalyticsService.kt

```diff
@@ -0,0 +1,60 @@
+package com.stripe.android.connect.analytics
+
+import android.app.Application
+import com.stripe.android.core.BuildConfig
+import com.stripe.android.core.Logger
+import com.stripe.android.core.networking.AnalyticsRequestV2
+import com.stripe.android.core.networking.AnalyticsRequestV2Factory
+import com.stripe.android.core.networking.DefaultAnalyticsRequestV2Executor
+import com.stripe.android.core.networking.DefaultStripeNetworkClient
+import com.stripe.android.core.networking.RealAnalyticsRequestV2Storage
+import com.stripe.android.core.utils.RealIsWorkManagerAvailable
+import kotlinx.coroutines.DelicateCoroutinesApi
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
+
+/**
+ * Service for logging [AnalyticsRequestV2] for the Connect SDK.
+ * This service is very simple. Consumers should prefer [ComponentAnalyticsService] instead,
+ * which uses this service internally.
+ */
+internal class ConnectAnalyticsService(application: Application) {
```

Comment:
Ideally this would be abstracted for testability. Imagine unit testing `EmbeddedComponentManager.onActivityCreate()`, which we should be doing (either now or after the public API is finalized).